config/model_configs/aquaplanet_rhoe_equil_clearsky_tvinsol_0M_slabocean.yml

```diff
@@ -0,0 +1,21 @@
+FLOAT_TYPE: "Float32"
+dt_save_state_to_disk: "10days"
+t_end: "10days"
+moist: "equil"
+precip_model: "0M"
+surface_setup: "DefaultMoninObukhov"
+vert_diff: "FriersonDiffusion"
+implicit_diffusion: true
+max_newton_iters_ode: 2
```

Comment:
Should we be using this one? `approximate_linear_solve_iters: 2`

Reply:
If you say so... some of the other long runs still use `max_newton_iters_ode: 2`. @szy21 , please can you advise?

Reply:
Yes, we should use `approximate_linear_solve_iters: 2`. If there are still long runs using `max_newton_iters_ode: 2` we should fix them:)

Reply:
All of these configs, or just long runs? 
![Screen Shot 2024-01-29 at 4 51 00 PM](https://github.com/CliMA/ClimaAtmos.jl/assets/38083334/f04f9df2-f58f-4650-963b-89c2cee1a6eb)

Reply:
(except the default?)

Reply:
Just longruns and `flame_perf_target_frierson`, but we can remove the ones that have 1 as that is the default. Don't worry about it for this PR. I can change it.